widget/embedded/src/pages/ConfirmSwapPage.tsx

```diff
@@ -38,15 +38,6 @@ import { useWalletsStore } from '../store/wallets';
 import { QuoteWarningType } from '../types';
 import { getContainer } from '../utils/common';
 
-const Container = styled('div', {
-  position: 'relative',
-  width: '100%',
-
-  '& .quote-update__alert': {
-    paddingTop: '$10',
-  },
-});
```

Comment:
If we eliminate styles for specific class names, we should also remove the associated class name from the corresponding element.

Reply:
fixed.